tools/cldr-apps/src/main/webapp/js/new/cldrRecentActivity.js

```diff
@@ -67,6 +75,224 @@ const cldrRecentActivity = (function () {
 
     return html;
   }
+
+  function showRecent(divName, locale, user) {
+    if (!locale) {
+      locale = "";
+    }
```

Comment:
locale is always null (then "")
divName is always "submitItems"
we can simplify later